queue_job/migrations/13.0.3.2.0/pre-migration.py

```diff
@@ -8,6 +8,8 @@
 
 
 def migrate(cr, version):
+    if not column_exists(cr, "queue_job", "record_ids"):
```

Comment:
I don't think this column will exist ever. It seems a one2many/many2many field.

Reply:
Sure, the problem is that the table does not exist